learn/what_is_meilisearch/telemetry.md

```diff
@@ -138,13 +138,18 @@ This list is liable to change with every new version of Meilisearch. It's not be
 | `requests.total_succeeded`              | Total number of successful search requests                                                  | 3456
 | `requests.total_failed`                 | Total number of failed search requests                                                      | 24
 | `requests.total_received`               | Total number of received search requests                                                    | 3480
-| `sort.with_geoPoint`                    | `true` if the sort rule `_geoPoint` was used, otherwise `false`                             | true
+| `sort.with_geoPoint`                    | `true` if the sort rule `_geoPoint` was specified, otherwise `false`                        | true
```

Comment:
You can ignore this, but the rest of the table uses "is specified".
```suggestion
| `sort.with_geoPoint`                    | `true` if the sort rule `_geoPoint` is specified, otherwise `false`                        | true
```